Kickstarter-iOS/Views/Controllers/ProjectPamphletContentViewController.swift

```diff
@@ -133,8 +138,9 @@ public final class ProjectPamphletContentViewController: UITableViewController {
   }
 
   fileprivate func goToComments(project: Project) {
-    let vc = DeprecatedCommentsViewController.configuredWith(project: project, update: nil)
-
+    let vc = self
+      .commentThreadingFeatureIsEnabled() ? CommentsViewController() : DeprecatedCommentsViewController
```

Comment:
We could just pull this out to an internal function in this file and add a test for the VC being returned based on the flag.

Reply:
Added it into the `ProjectPamphletContentViewControllerTests` file. I'm running an assertion on the `Type` and not any snapshots. Let me know if that's the right location for the test.